docs/install/file-storage/README.md

```diff
@@ -0,0 +1,115 @@
+# FlowForge File Storage
+
+When running in cloud like environment (under Docker or Kubernetes)
+FlowForge projects do not have access to a persistent filesystem to 
+use to store files. Files written to the container file system will 
+lost if the project is restarted. 
+
+For this reason we recomend that the Node-RED core file nodes are
+disabled on these platforms.
+
+To solve this limitation FlowForge provides a set of replacement
+file nodes. These are backed by Object Store which can be configured
+to use a mounted volume or S3 bucket.
+
+The FlowForge File Storage system can also be used with the FlowForge
+LocalFS build, but in this case the FlowForge Projects already have
+direct access to the file system of the machine hosting FlowForge.
+
+## Configuring
+
+The Docker Compose and Kubernetes Helm Chart will both now start a 
+container running the File Server application.
+
+There are 2 main things to set up:
+
+ 1. Which backend to use (`localfs` or `s3`)
+ 2. The configuration options for the backend
+
+### LocalFS
+
+This can be used when you mount a volume into the File Server container
+to persist files.
+
+The only configuration option used is the path to the directory to use
+as the root of the storage
+
+```
+host: 0.0.0.0
+port: 3001
+base_url: http://flowforge:3000
+driver:
+  type: localfs
+  options:
+    root: var/root
+```
+
+### S3
+
+The following can be any of the options for the S3Client Contructor, see [here](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/s3clientconfig.html)
```

Comment:
The linked doc isn't approachable I feel, only JS devs might but will not always understand it. Further; it's very AWS specific in the words and phrases used. 

What about:

```
Option        | Description
--------------|------------
`bucket`  | **required** name of the S3 Bucket
`region` | Region the for data residency

etc, etc